lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -195,12 +195,17 @@
 '''
 
 RETURN = '''
-vm:
-    description:
-      - The virtual machine that user manages.
-      - It contains all attributes https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachine
-    returned: success
-    type: complex
+kubevirt_vm:
+  description:
+    - The virtual machine managed by the user.
+  returned: success
+  type: complex
+  contains:
+    k8s_objects:
+      description:
+      - Lists kubernetes objects this module created, modified or read during operation, by type name.
```

Comment:
It doesn't currently return, the VMI. But I wouldn't hold this PR anymore, let's merge and create issue to fix it.